src/commands/src/helpers/index.ts

```diff
@@ -166,24 +168,29 @@ export async function executeCliCommand({
 
     if (writeStream) executedCommand.stdout?.pipe(writeStream);
     executedCommand.stdout?.on('data', (data: Buffer) => {
+      if (logger) logger(data.toString());
```

Comment:
This is a nice addition! I mentioned in a previous comment that I would describe how we would change this. Instead of using the `logger` function, let's just use a plain old `console.log` so that we aren't writing to the database. We won't get the real-time result from the dashboard, but in AWS CloudWatch, the `console.log` statements will show up in real time.

Reply:
Alright, so the logs will be very short and possibly not have much logging to show if there's been an issue with a "successful" command output. Do we think that's worth the risk? It's the only way we caught multiple issues in this process, honestly

Reply:
We can still log the result after it has finished completely. For example:

```ts
  const result = await executeCliCommand({});
  
  this.logger.save(..., result.outputText);
  this.logger.save(..., result.errorText);

```

For this, we are looking to avoid calling `logger.save` which uploads the logs to Atlas every time. The `stdout.on('data')` `stderr.on('data')` get called frequently, so this would lead to us uploading to Atlas much more often.